modules/installation-alibaba-ram-user.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-account.adoc
+
+[id="installation-alibaba-ram-user_{context}"]
+= Creating a RAM user
+
+.Questions
+
+. When configuring the access mode, is `API Call-based Access` a required access mode?
+
+. While we have a list of permissions for the cloud account, is there a specific out-of-the-box policy that can be assigned to the RAM user or must a policy be created?
```

Comment:
@mtulio @jianli-wei @kwoodson @joelddiaz
Can you help with these questions?

Reply:
There is not an out-of-the-box policy that encompasses every one of the permissions other than the `AdministratorAccess` policy. This is the super user permission.